.gitlab/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ["1.15", "1.14", ""]
+    name: Test ${{ matrix.go }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.35.0
+      - run: go test -v
```

Comment:
Do we need this file? I guess we only need `.github/workflows/ci.yml` right?

Reply:
Yes. I should have renoved this : facepalm: